src/analytics.ts

```diff
@@ -17,7 +18,16 @@ export function createAnalyticsInstance(options?: Options) {
         event_cache: Array<{ event: keyof TEvents; payload: TEvents[keyof TEvents] }> = [],
         page_view_cache: Array<{ current_page: string; platform: string; user_id: string }> = []
 
-    const initialise = ({ growthbookKey, growthbookDecryptionKey, rudderstackKey, growthbookOptions }: Options) => {
+    const initialise = async ({
+        growthbookKey,
+        growthbookDecryptionKey,
+        rudderstackKey,
+        growthbookOptions,
+    }: Options) => {
+        const response = await fetch('https://www.cloudflare.com/cdn-cgi/trace')
+        const text = await response?.text()
+        const CloudflareCountry = Object.fromEntries(text.split('\n').map(v => v.split('=', 2))).loc.toLowerCase()
+        console.log('CloudflareCountry', CloudflareCountry)
```

Comment:
please remove this console log
```suggestion
        console.log('CloudflareCountry', CloudflareCountry)
```